### ISSUE TITLE
python:3.5.1 image is broken

### ISSUE THREAD

**Issue by jessy1092** [docker-library/python#68]
I use python:3.5.1 image, but it didn't have `python` and `python3` command
```shell
[root@client ~]# docker run -i -t python:3.5.1 /bin/bash
root@b5df006d1d78:/# python
bash: python: command not found
root@b5df006d1d78:/# python3
bash: python3: command not found
```

python:3.5.0 is ok
```shell
[root@client ~]# docker run -i -t python:3.5.0 /bin/bash
root@f5dc0c6d5a04:/# python
Python 3.5.0 (default, Dec  5 2015, 22:14:04)
[GCC 4.9.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 
```

Images info
```shell
[root@client ~]# docker images
REPOSITORY                                 TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
python                                     3.5                 cd86406b3214        3 hours ago         688.7 MB
python                                     3.5.1               cd86406b3214        3 hours ago         688.7 MB
python                                     3.5.0               c64596cbb9b6        3 days ago          688.8 MB
```

**Comment by yosifkit** [comment]
I am unable to reproduce this. I did a fresh `docker pull`. 
```console
$ docker images python
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
python              3.5.1               7a7c8d68b039        22 hours ago        688.7 MB
$ docker run -it --rm python:3.5.1 bash
root@6f691637a49b:/# python --version
Python 3.5.1
root@6f691637a49b:/# 
```

Can you try pulling again?  If that doesn't work, can you post the output of `uname -a`, `docker version`, and `docker info`?

**Comment by jessy1092** [comment]
@yosifkit 

Magic! It's work after I deleting 3.5.1 image and pulling again. Thank for help.

I don't know which process was wrong. But it's OK now.

BTW, my information. Maybe could find out what's wrong.
uname -a
```
Linux client 2.6.32-504.el6.x86_64 #1 SMP Wed Oct 15 04:27:16 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
```

docker version
```
Client version: 1.7.1
Client API version: 1.19
Go version (client): go1.4.2
Git commit (client): 786b29d
OS/Arch (client): linux/amd64
Server version: 1.7.1
Server API version: 1.19
Go version (server): go1.4.2
Git commit (server): 786b29d
OS/Arch (server): linux/amd64
```

docker info
```
Containers: 21
Images: 223
Storage Driver: devicemapper
 Pool Name: docker-253:0-2753325-pool
 Pool Blocksize: 65.54 kB
 Backing Filesystem: extfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 7.748 GB
 Data Space Total: 107.4 GB
 Data Space Available: 38.49 GB
 Metadata Space Used: 13.68 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.134 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.95-RHEL6 (2015-09-08)
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 2.6.32-504.el6.x86_64
Operating System: <unknown>
CPUs: 8
Total Memory: 7.404 GiB
Name: client
ID: Z5P2:IO5M:5ICA:KWZI:MNWC:SZHO:ACNG:2O2E:YUKY:RQX7:SXHS:FS5R
```

**Comment by yosifkit** [comment]
My best guess is that it was somehow a bad pull. :confused:   Let us know if you have any issues again.
